protocol/xmrmaker/instance.go

```diff
@@ -142,7 +142,7 @@ func (inst *Instance) createOngoingSwap(s *swap.Info) error {
 
 	ethSwapInfo, err := inst.backend.RecoveryDB().GetContractSwapInfo(s.ID)
 	if err != nil {
-		return fmt.Errorf("failed to get info for ongoing swap, id %s: %s", s.ID, err)
+		return fmt.Errorf("failed to get contract info for ongoing swap, id %s: %s", s.ID, err)
```

Comment:
Should we change that second `%s` to `%w`? I'm not sure adding the word `contract` in isolation clarifies things. It makes it sound like it was an ethereum related error.  Maybe this?
`failed to load contract info for ongoing swap from db, swapID %s: %w`